repoqa/search_needle_function.py

```diff
@@ -156,6 +164,7 @@ def make_cache_id(lang, repo, needle_name, code_context_size, position_ratio):
 
 def evaluate_model(
     model: str,
+    is_base_model: bool = False,
```

Comment:
Maybe we can detect this automatically:
1. for HF models we can check template
2. for non-HF models I thin they are all chat models (e.g., from oai, gemini, etc)